spec/board_spec.rb

```diff
@@ -173,16 +153,26 @@
     describe 'play' do
         context 'when X winning sequence played' do
             it 'winner returns X' do
-                board = Board.new()
-                board.play(1)
-                board.play(2)
-                board.play(4)
-                board.play(5)
-                board.play(7)
+                board = make_board_and_play_moves([1, 2, 4, 5, 7])
                 expect(board.winner()).to eq('X')
             end
         end
     end
+
+    describe 'to_s' do
+        context 'to_s called' do
+            it 'returns string representation' do
+                board = make_board_and_play_moves([7, 8, 5, 6])
+                expect(board.to_s).to eq("---\n-XO\nXO-\n")
+            end
+        end
+    end
+end
+
+def make_board_and_play_moves(moves)
```

Comment:
Not sure about this method name. It's very literal as to what the method does. Then you almost need to say `make_board_and_play_moves_and_return_board_afterwards` 😹 
so maybe just call it `board_with_moves`